docs/design-docs/02-custom-root-reconciler-clusterrole.md

```diff
@@ -1,15 +1,21 @@
 # Custom `ClusterRole` assignments for `RootSync`s
 
-* Author(s): @tomasaschan
-* Approver: @sdowell, @janetkuo
+* Author(s): @tomasaschan, @sdowell
+* Approver: @mortent, @karlkfi
 * Status: approved
 
 ## Summary
 
 When creating a reconciler deployment for a `RootSync`, Config Sync currently [creates a
 `ClusterRoleBinding`] for the generated `ServiceAccount` granting it `cluster-admin`.
 
-This proposal introduces a mechanism to customize this behavior.
+On the other hand, when creating a reconciler deployment for a `RepoSync`, Config Sync
+currently creates `RoleBinding` for the generated `ServiceAccount` granting
+it `configsync.gke.io:ns-reconciler`. This default `ClusterRole` grants basic permissions required for the `RepoSync`
+reconciler to function, but it's up to users to manage additional `RoleBinding`s.
+
+This proposal introduces a way to manage the lifecycle of RBAC bindings for both
```

Comment:
I would move this to the top. It's the actual summary. The other stuff above is more like historical context, not summary.

Reply:
done